src/UI/Component/Button/Tag.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/* Copyright (c) 2017 Nils Haagen <nils.haagen@concepts-and-training.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Button;
+
+use \ILIAS\UI\Component\Counter\Counter;
+
+/**
+ * This describes a tag(-button).
+ */
+interface Tag extends Button {
+	const RELLOW  		= 'btn-tag-relevance-verylow';
+	const RELVERYLOW 	= 'btn-tag-relevance-low';
+	const RELMID  		= 'btn-tag-relevance-middle';
+	const RELHIGH  		= 'btn-tag-relevance-high';
+	const RELVERYHIGH  	= 'btn-tag-relevance-veryhigh';
+
+	/**
+	 * Set relevance of Tag (to distinguis visually)
+	 *
+	 * @param	int	 $relevance  a value between 1 and 5
+	 * @throws 	\InvalidArgumentException
+	 * @return	Tag
+	 */
+	public function withRelevance($relevance);
+
+	/**
+	 * Get the relevance of the Tag.
+	 *
+	 * @return	int
+	 */
+	public function getRelevance();
+
+	/**
+	 * Get CSS-class according to the relevance of the Tag.
+	 *
+	 * @return	string
+	 */
+	public function getRelevanceClass();
+
+	/**
+	 * Set a fix background-color.
+	 *
+	 * @param	Color $col
+	 * @return	Tag
+	 */
+	public function withBackgroundColor($col);
```

Comment:
We have cases where tags are system-induced, i.e. where trainings are tagged with categories.
Those categories are prone to changes, so certain users can adjust and extend categories. This includes coloring. 
Although I'm all in for classes, this would mean adding CSS for every new category, so I have this parallel mechanism where colors can be stored in the database instead. I think, both ways can easily live next to each other, with the CSS-classes as the default way of using tags.